src/resources/Tailgate/TailgateCatalogContent/CatalogContent.ts

```diff
@@ -0,0 +1,18 @@
+import Resource from '../../Resource.js';
+import API from '../../../APICore.js';
+
+export default class CatalogContent extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/catalogcontent/source/`;
+
+    getObjectTypes(sourceId: string) {
+        return this.api.get(`${CatalogContent.baseUrl}/${sourceId}/objecttypes`);
+    }
+
+    getMetadataValues(sourceId: string, objectType: string) {
+        return this.api.get(this.buildPath(`${CatalogContent.baseUrl}/${sourceId}/metadatavalues`, objectType));
+    }
+
+    getMetadata(sourceId: string, objectType: string) {
+        return this.api.get(this.buildPath(`${CatalogContent.baseUrl}/${sourceId}/metadata`, objectType));
```

Comment:
Here objectType should be an object :point_right: `{objectType}`

Reply:
fixed moved to object ObjectType